.travis.yml

```diff
@@ -0,0 +1,50 @@
+dist: xenial
+
+language: php
+
+before_install:
+  - nvm install
+  - nvm use
+
+install:
+  - npm install
+
+jobs:
+
+  include:
+
+    - php: "5.6"
+      script:
+        - npm run lint || true
+        - npm run build
+
+    - php: "7.4"
+      script:
+        - npm run lint || true
+        - npm run build
+
+    - stage: deploy
+      php: "5.6"
+      name: Package
+      script:
+        - npm run build
+        - cp cloudinary-image-management-and-manipulation-in-the-cloud-cdn.zip "cloudinary-image-management-and-manipulation-in-the-cloud-cdn-$TRAVIS_TAG.zip"
+      deploy:
+        provider: releases
+        repo: kasparsd/cloudinary_wordpress
```

Comment:
TODO: Change to the current repo. This ensure we don't try to publish releases on forked repositories.